projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-universal.php

```diff
@@ -333,19 +352,20 @@ function ( $payment_gateway ) {
 				$this->record_event(
 					'woocommerceanalytics_product_checkout',
 					array(
-						'pq'                => $cart_item['quantity'],
-						'payment_options'   => $enabled_payment_options,
-						'device'            => wp_is_mobile() ? 'mobile' : 'desktop',
-						'guest_checkout'    => 'Yes' === $guest_checkout ? 'Yes' : 'No',
-						'create_account'    => 'Yes' === $create_account ? 'Yes' : 'No',
-						'express_checkout'  => 'null',
-						'shipping_option'   => $this->get_shipping_option_for_item( $cart_item_key ),
-						'products_count'    => $products_count,
-						'coupon_used'       => $coupon_used,
-						'order_value'       => $order_value,
-						'store_currency'    => get_woocommerce_currency(),
-						'additional_blocks' => $this->additional_blocks_on_page,
-						'template_used'     => $this->cart_checkout_templates_in_use ? '1' : '0',
+						'pq'                             => $cart_item['quantity'],
+						'payment_options'                => $enabled_payment_options,
+						'device'                         => wp_is_mobile() ? 'mobile' : 'desktop',
+						'guest_checkout'                 => 'Yes' === $guest_checkout ? 'Y' : 'N',
+						'create_account'                 => 'Yes' === $create_account ? 'Y' : 'N',
+						'express_checkout'               => 'null',
+						'shipping_option'                => $this->get_shipping_option_for_item( $cart_item_key ),
+						'products_count'                 => $products_count,
+						'coupon_used'                    => $coupon_used,
+						'order_value'                    => $order_value,
+						'store_currency'                 => get_woocommerce_currency(),
+						'additional_blocks_on_cart_page' => $this->additional_blocks_on_cart_page,
+						'additional_blocks_on_checkout_page' => $this->additional_blocks_on_checkout_page,
+						'template_used'                  => $this->cart_checkout_templates_in_use ? '1' : '0',
```

Comment:
A lot of those seems to be shared between events, can we move them all to a single place so we don't make mistakes?